src/modules/common/TabPanel.tsx

```diff
@@ -0,0 +1,32 @@
+import { styled } from '@mui/material';
+import Box from '@mui/material/Box';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+const TabPanelDiv = styled('div')({
+  flexGrow: 1,
+  width: '100%',
+});
+
+const TabPanel = (props: TabPanelProps): JSX.Element => {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <TabPanelDiv
+      role="tabpanel"
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      aria-labelledby={`tab-${index}`}
+      {...other}
+    >
+      {/* TODO: Fix height problem with tabs. */}
```

Comment:
Is this still an issue ?

Reply:
Not sure. I don't remember fixing it but I cannot reproduce it now. It may depend on the screen resolution.